redaxo/src/addons/media_manager/lib/effect_abstract.php

```diff
@@ -56,14 +59,17 @@ public function getName()
      *       // ... the next input-field spec
      *     ];
      *
-     * @return array
+     * @return list<array{label: string, name: string, type: 'int'|'float'|'string'|'select'|'media', default?: mixed, notice?: string, prefix?: string, suffix?: string, attributes?: array, options?: array}>
      */
     public function getParams()
     {
         // implement me in your subclass.
         return [];
     }
 
+    /**
+     * @param resource $gdImage
```

Comment:
In php8 vermutlich keine resource mehr,.. weiß aber net wie/ob wir damit umgehen müssen

Reply:
Guter Punkt, den ich nicht mehr auf dem Schirm hatte.
Allerdings weiß ich auch nicht, wie wir damit umgehen. Wenn ich da die `GdImage`-Klasse ergänze, hagelt es an Fehlern.
Ich denke, ich würde es in den docs erstmal bei resource lassen, und habe mir dieses Issue mal auf die Beobachtung gesetzt: https://github.com/vimeo/psalm/issues/3824